src/infrastructure/gateway/transaction/TransactionGateway.ts

```diff
@@ -1,13 +1,10 @@
-/* eslint-disable */
```

Comment:
removed unused import and no warnings without the eslint-disable, perhaps the cause has since been removed